src/core_atmosphere/physics/physics_wrf/module_sf_urban.F

```diff
@@ -2131,6 +2163,11 @@ SUBROUTINE urban_param_init(DZR,DZB,DZG,num_soil_layers, &
             if(allocate_status /= 0) FATAL_ERROR('Error allocating PERFLO_TBL in urban_param_init')
             ALLOCATE( HSESF_TBL(ICATE), stat=allocate_status )
             if(allocate_status /= 0) FATAL_ERROR('Error allocating HSESF_TBL in urban_param_init')
+            ALLOCATE( PV_FRAC_ROOF_TBL(ICATE), stat=allocate_status )
+            if(allocate_status /= 0) CALL wrf_error_fatal('Error allocating PV_FRAC_ROOF_TBL in urban_param_init')
```

Comment:
Need to change to FATAL_ERROR call on this line and two lines below.

Reply:
I rewrote those lines  separately with proper indentation to make sure about all the if statements. I am kind of confused too. Will send an email to Michael.

Reply:
@ldfowler58 I believe you still have to make change here. Even though urban options are not supported yet, and maybe this part of the code is not compiled, wrf_error_fatal doesn't exist in MPAS code. Per our early decision to redefine them at the top of the routine:
```
#if defined(mpas)
use mpas_atmphys_date_time, only: cal_mon_day
use mpas_atmphys_utilities, only: physics_error_fatal
#define FATAL_ERROR(M) call physics_error_fatal( M )
#else
 use module_ra_gfdleta,  only: cal_mon_day
 use module_wrf_error
#define FATAL_ERROR(M) call wrf_error_fatal( M )
#endif
```
we should use FATAL_ERROR instead of calling wrf_error_fatal in WRF or physics_error_fatal in MPAS. I'm making a PR on WRF side to make this change too. It should be there in 4.5.1.